lua/starfall/libs_sh/http.lua

```diff
@@ -58,8 +80,8 @@ function http_library.get(url, callbackSuccess, callbackFail, headers)
 		end
 	end
 
-	if requests[instance.player] then SF.Throw("You can't run a new http request yet", 2) end
-	requests[instance.player] = true
+	if not http_library.canRequest() then SF.Throw(string.format("You have hit the maximum amount of concurrent HTTP requests (%d)", http_max_active:GetInt()), 2) end
```

Comment:
Can't use http_library.canRequest since the user can replace that with `function() return true end`

Reply:
Sorry, didn't think about that